docs/extensions/kuneiform-precompiles/precompiles.mdx

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_position: 1
+sidebar_label: "Precompiles"
+id: precompiles
+title: Kuneiform Precompiles
+description: Extend Kuneiform with custom precompiles.
+slug: /extensions/precompiles
+---
+
+Kuneiform precompiles are custom libraries that can be imported and called in Kuneiform actions.
+By implementing custom action precompiles, developers can add support for new functionality to their Kwil network's Kuneiform language.
+These functionalities are wide ranging, and can include anything from performing hashing algorithms to directly calling the active Postgres connection.
+
+## How Precompiles Work
+
+Action precompiles are implemented as Go libraries that are loaded into the Kwil node at startup. All precompiles are globally available to all Kuneiform schemas.
+There are two distinct steps in an action precompile's lifecycle:
+
+1. **Initialize**: The action precompile is imported into a Kuneiform schema. It is given a set of initialization values, and a name by which it can be called. Initialize is called
```

Comment:
```suggestion
1. **Initialize**: The action precompile is imported into a Kuneiform schema. It is given a set of initialization values, and an alias by which it can be called. Initialize is called
```

I feel like alias more clearly indicates this is how it is being named in kuneiform, rather than the name it is given in the `RegisterExtension` call?